Vitals/AppDelegate.swift

```diff
@@ -10,25 +10,18 @@ import SwiftUI
 
 @main
 class AppDelegate: NSObject, NSApplicationDelegate {
-    var popover = NSPopover.init()
     var statusBar: StatusBarController?
     let processMonitor = ProcessMonitor()
 
     func applicationDidFinishLaunching(_ aNotification: Notification) {
         processMonitor.start()
         
         // Create the SwiftUI view that provides the contents
-        let viewModel = ContentViewModel(monitor: processMonitor, contentVisible: false)
+        let viewModel = ContentViewModel(monitor: processMonitor, contentVisible: true)
```

Comment:
The `contentVisible` property can never be false anymore. Thoughts on removing it?

Reply:
Thanks for pointing this out!

I actually chose to fix it up rather than remove it (11b12dd). Without it, the SwiftUI ContentView gets re-rendered every second, even when the menu bar is closed, and that eats a bit of unnecessary CPU. If you know a better way of handling this, I'm all ears though 😄